spark-derived-feature-view/docker-compose.yaml

```diff
@@ -0,0 +1,21 @@
+version: "2.1"
+services:
+  spark:
+    image: feathub-spark:latest
+    container_name: spark
+    ports:
+      - "8080:8080"
+      - "7077:7077"
+    environment:
+      - SPARK_WORKER_CORES=1
+      - SPARK_WORKER_MEMORY=1G
+      - SPARK_DRIVER_MEMORY=1G
+      - SPARK_EXECUTOR_MEMORY=500M
+    volumes:
+      - ./main.py:/tmp/main.py
+      - ./data:/tmp/spark-data
```

Comment:
Would it be better to use `/tmp/data` for consistency with other examples?

Reply:
According to the above discussion, we need to tell users to copy data files to the same directory in the container before running `main.py`. And the `/tmp/data` is kind of a common-used directory, so I prefer to use `/tmp/feathub-data` to avoid overwriting user files, what do you think?

Reply:
Sounds good.